pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -314,6 +314,8 @@ self: super: {
   # defined as >= 3.1  && < 3.2, can be removed once pandoc gets bumped by Stackage.
   patat = super.patat.override { pandoc = self.pandoc_3_1_6; };
 
+  twain = super.twain.override { http2 = self.http2_3_0_3; };
```

Comment:
Can you add a comment that this override is repeated in all-packages.nix so that we don’t forget to remove it there as well?

Reply:
Done.